app/src/main/java/com/globant/example/mentorapp/home/presentation/view/adapter/ListUsersAdapter.java

```diff
@@ -10,41 +10,41 @@
 
 import com.globant.example.mentorapp.R;
 import com.globant.example.mentorapp.home.presentation.model.ModelUserEntity;
-import com.globant.example.mentorapp.home.presentation.view.activity.HomeScreenActivity;
 import com.globant.example.mentorapp.home.presentation.view.fragment.ListUsersViewFragment;
 import com.globant.example.mentorapp.util.Utilities;
 import com.squareup.picasso.Picasso;
 
 import java.util.List;
 
 /**
- * Custom adapter to show General data fro users.
+ * Custom adapter to show General data from users.
  * Created by oscar.amezquita on 8/06/2017.
  */
 
 public class ListUsersAdapter extends RecyclerView.Adapter<ListUsersAdapter.ViewHolder> {
 
     private List<ModelUserEntity> users;
     private Context context;
-    private ListUsersViewFragment fragmentListener;
+    private ListUsersViewFragment.OnUserClickListener onUserClickListener;
```

Comment:
Excuse me if I confused you regarding this. My point is OnUserClickListener should be an interface define in this class and implemented by your Fragment. And inside your fragment you just need to pass `this` as parameter in the adapter constructor.